### PR TITLE
Clarify information about external graders

### PR DESCRIPTION
**Pull request by srpearce**
## [DOC-3893](https://openedx.atlassian.net/browse/DOC-3893)

Per request from partner support, clarify general and setup information about external graders.

**Note** The failing tests are due to an unrelated file (this is a fairly old branch). Merging this PR shouldn't cause any problems.

### Reviewers

Possible roles follow. The PR submitter checks the boxes after each reviewer finishes and gives :+1:. 

- [ ] Subject matter expert: @jibsheet 
- [ ] Doc team review (copy edit): @edx/doc
- [ ] Partner support: @enavarro

FYI: Tag anyone else who might be interested in this PR here.

### HTML Version

To see a built version of these changes, go to https://draft-xqueue-information-update-jun-2018.readthedocs.io/en/latest/exercises_tools/external_graders.html

### CODE REVIEW

**Review by grantgoodmanedX — APPROVED** [review]
👍 Looks good to me

**Comment by enavarro** [comment]
Looks amazing! 👍 so happy to see this going live!

File: en_us/shared/exercises_tools/external_graders.rst [hunk 1]
```diff
@@ -31,285 +51,302 @@ An external grader is particularly useful for software programming courses
...
+prompt. Replace the placeholder values with the values for your credentials.
 
-.. _The XQueue Interface:
+::
 
-**************************************
-The XQueue Interface
-**************************************
+  curl -v -d "username=&password=" "https://xqueue.stage.edx.org/xqueue/login/"
+  curl -v -b "sessionid=returnedbylogin"   "https://xqueue.stage.edx.org/xqueue/get_queuelen/?queue_name=your_q"
```
Comment (jibsheet) [inline en_us/shared/exercises_tools/external_graders.rst]: We should strike stage.  Most of our partners don't have access or courses there to test unfortunately.

**Review by jibsheet — APPROVED** [review]
Only minor change requested

**Pull request merged**